sqs/src/test/java/akka/stream/alpakka/sqs/javadsl/SqsSourceTest.java

```diff
@@ -77,4 +79,29 @@ public void streamFromQueue() throws Exception {
         assertEquals(input.size(), cs.toCompletableFuture().get(3, TimeUnit.SECONDS).size());
 
     }
+
+    @Test
+    public void streamFromQueueWithCustomClient() throws Exception {
+
+        //#init-custom-client
+        final ExecutorService executor = Executors.newFixedThreadPool(10);
+        final AmazonSQSAsyncClient sqsClient = new AmazonSQSAsyncClient(credentials, executor)
+            .withEndpoint("http://localhost:9324");
+        //#init-custom-client
+
+        final String queueUrl = randomQueueUrl();
+
+        final List<String> input = IntStream.range(0, 100).boxed().map(i -> String.format("alpakka-%s", i)).collect(Collectors.toList());
```

Comment:
The entire thing could just be:

`Source.range(0, 100).runForeach(i -> String.format("alpakka-%s", i), mat)` 

😉 

Could you replace that? Much simpler IMO